torch/testing/_internal/common_jit.py

```diff
@@ -0,0 +1,168 @@
+# Torch
+import torch
+import torch.cuda
+import torch.jit
+import torch.jit._logging
+import torch.jit.frontend
+import torch.jit.quantized
+
+# Testing utils
+from torch.testing import floating_and_complex_types
+from torch.testing._internal.common_utils import TestCase, \
+    freeze_rng_state, TemporaryFileName, enable_profiling_mode_for_profiling_tests
+from torch.testing._internal.common_utils import enable_profiling_mode  # noqa: F401
+
+# Standard library
+from itertools import chain
+
+import io
+
+def check_output_types(self, func, ref_outputs, args, kwargs):
```

Comment:
This file was created from pulling functions and methos from test_jit.py and jit_utils.py so that the test_ops.py test suite can re-use the same functions (I was trying to reduce code duplication)